README.md

```diff
@@ -248,6 +248,39 @@ module.exports = {
 }
 ```
 
+### Locales
+
+Often we render multiple versions of our application for different locations. Eg. Australia & New Zealand. To render an html file for each location you can use the locales option in `sku.config.js`.
+
+Locales accepts an array of strings representing each locale you want to render html files for.
```

Comment:
When referencing an option/argument name, we should use `locales` instead of "Locales". Also, you'll need to re-order this sentence so it's not the first word, otherwise it'll look weird.

(Also need to update to "HTML" here too)